package.json

```diff
@@ -8,6 +8,7 @@
     "preact": "8.2.5",
     "preact-compat": "3.17.0",
     "ramda": "0.25.0",
+    "seedrandom": "^2.4.3",
```

Comment:
we can go ahead and lock this down (remove the `^`).

Reply:
Changed this in: 66fbdb0